webrtc.nimble

```diff
@@ -24,10 +24,18 @@ var cfg =
   " --threads:on --opt:speed"
 
 when defined(windows):
-  cfg = cfg & " --clib:ws2_32"
+  cfg = cfg & " --clib:ws2_32 --clib:iphlpapi"
```

Comment:
It might be good to add a comment explaining why this is necessary.

Reply:
Done.